.github/workflows/ci.yml

```diff
@@ -146,4 +146,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: '14'
+      # The install step has been added here such that the `.yarn/install-state.gz` file is generated. This file is used
+      # by the script `check-licenses` below.
+      - run: yarn install --immutable
       - run: yarn check-licenses
```

Comment:
Running `yarn check-licenses` locally will succeed because it generates the state file by itself. This however does not seem to happen in the CI, this is why this `yarn install` was added.

Reply:
This could potentially be replaced by a Python version of the license checker (e.g. in the worker repo).